### ISSUE TITLE
508-defect-2 [HEADINGS]: Headings should be logically ordered on the upload page

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45029]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh Kim @joshkimux 

## User Story or Problem Statement

As a screen reader user, I use headings to:

- Understand the structure of a page
- Find out what sections exist
- Navigate to sections I'm interested in

I expect headings to be logically ordered and consistent with the visual structure of the page.

## Details

Heading order on the upload page has a couple of issues to be addressed:
- Upload your documents should be a `h3` given its a child of the `h2` "Step x out of y"
- Having problems uploading... should also be a `h3` given its styled as a heading and starts a new subsection on the page

<img width="1371" alt="Screen Shot 2022-07-28 at 12 07 15 PM" src="https://user-images.githubusercontent.com/14154792/181588994-36b2a668-9975-4b12-9156-12bef65c4e64.png">

## Acceptance Criteria

- [ ] Upload your documents is a `h3`
- [ ] Having problems uploading is a `h3`

**Comment by obliviga** [comment]
@joshkimux, this issue should be addressed by the COE / LGY team. I'm not sure what either of those acronyms stand for, but that's what the application is labelled in our repo. Since this is not a forms library issue, I'm going to unassigned myself from this ticket.

For the engineer on COE / LGY who looks at this ticket, I believe the issues reside in the following files:

- `src/applications/lgy/coe/form/config/chapters/documents/UploadRequirements.jsx`
- `src/applications/lgy/coe/form/config/chapters/documents/fileUpload.js`

**Comment by joshkimux** [comment]
Thanks so much @obliviga , appreciate the help as always!

**Comment by Mottie** [comment]
These fixes are now available in staging for review.

**Comment by joshkimux** [comment]
![Uploading Screen Shot 2022-08-19 at 3.46.35 PM.png…]()
Confirming this has been fixed, you rock @mottie

**Comment by joshkimux** [comment]
![Uploading Screen Shot 2022-08-19 at 3.46.35 PM.png…]()
Confirming this has been fixed, you rock @mottie
